src/main/java/org/assertj/core/api/InstanceOfAssertFactories.java

```diff
@@ -0,0 +1,70 @@
+package org.assertj.core.api;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import java.util.function.DoublePredicate;
+import java.util.function.IntPredicate;
+import java.util.function.LongPredicate;
+import java.util.function.Predicate;
+
+/**
+ * Static {@link InstanceOfAssertFactory InstanceOfAssertFactories} for {@link Assert#instanceOf(InstanceOfAssertFactory)}.
+ *
+ * @since 3.13.0
+ */
+public final class InstanceOfAssertFactories {
+
+  public static final InstanceOfAssertFactory<Predicate, PredicateAssert<Object>> PREDICATE = new InstanceOfAssertFactory<>(Predicate.class,
```

Comment:
there is warning `Predicate is a raw type. References to generic type Predicate<T> should be parameterized` but I'm not sure this is easily solvable, if not, suppress the warning and add a comment explaining why this is ok.